docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-configure-dex.md

```diff
@@ -0,0 +1,1336 @@
+---
+title: How to configure Data Exchange API endpoints for import or export data. 
```

Comment:
Everything except `Validation constrains` are already part of this existing documentation https://docs.spryker.com/docs/pbc/all/data-exchange/202404.0/configure-data-exchange-api.html#create-and-configure-a-data-exchange-api-endpoint

1. Why do we need all these example configurations for discounts, vouchers, etc.?
2. If the answer to the first question is that it is just a generic how-to page, please create a separate how-to page `Configure Data Exchange API validation`, and get rid of this one.